reviewer-guide.en.md

```diff
@@ -0,0 +1,71 @@
+- If you didn't update your rating or comments, or at least entering the same rating or comments, **your rating given in first round will not be taken into consideration**. (Except which didn't modifiy in the modify stage)
+- Now you can view **the information of the author**, and **other reviewers' rating and comments (anonymous)**
+- After the end of this round, it will determine the accepted proposal. If the accepted proposal is insufficient or exceed the total number, the Program Committee will make the decision.
+
+### Review Announcement
+After review, we will email the second round rating and comments to authors. If you don't want to announce your rating & comments, please check the box of "cancel sending comments" optino.
+
+# Review Notices
++ Given **constructive comments** can help the review process, and help author in the modify stage.
++ When rating, please put yourself into author's shoes,  refer to [《How to Propose a Talk》](https://tw.pycon.org/2018/en-us/speaking/talk/) about good ideas and bad ideas as judging criteria. We also welcome our reviewer to give us more advice for how to distinguish good and bad ideas in the proposal.
++ Please based on **Python Level** of the proposal to rating and comments.
+For Python Level, please refer to [《How to Choose the Python Level》](https://tw.pycon.org/2018/en-us/speaking/talk/)。
++ Although we are the reviewer, accept or not is our rating standard. But for PyCon TW, is not only accept or not, but also to enpower the community. Please judge as **how to make the proposal better**, not as a judger to judge in a sentence or not.
++ Even the review process is anonymous, we should respect author and follow PyCon TW [Code of Conduct](https://tw.pycon.org/2018/en-us/about/code-of-conduct//).
+
+# Review Detail
++ Proposal language have: English talk, Chinese talk with English slide, Chinese talk with Chinese slide, Taiwanese Hokkien. When author choose to use **English talk**, please use english to comments.
+
+
+# Rating and Accept mechanism
+### Rating
+
+Rating only have four level: -1, -0, +0, +1.
+
+Each rating level's meaning:
+* **-1**：Strong Reject    This **SHOULD NOT** be in PyCon
+* **-0**：Weak Reject
+* **+0**：Weak Accept
+* **+1**：Strong Accept    This **SHOULD** be in PyCon
+
+### Accept mechanism
+
+When each round ends, some of the proposal will be reject or accept, the standard is:
+* **Reject**：one or more -1, no any +0, +1
+* **Accept**：one or more +1, no any -0, -1
+
+Here are some examples:
+* **Talk #1** Rejected
+  -1, -0, -0, -0, -1
+* **Talk #2** Do nothing, moved to stage #2
+  -1, -0, -0, +0
+* **Talk #3** Accepted
+  +1, +0, +1, +0, +0, +0, +0
+* **Talk #4** Do nothing, moved to stage #2
+  -1, +1, -0, -0
+
+
+# Q & A
+**Q**: If in some reason, I may know or I may guess out some proposal's author. In the first round, this may cause some conflict of interest, do I qualify to review this proposal?
```

Comment:
```suggestion
**Q**: If for some reason, I may know or guess the author of a proposal. In the first round, this may cause some conflict of interest, do I qualify to review this proposal?
```